openmc/mesh.py

```diff
@@ -870,6 +1050,12 @@ def z_grid(self):
     def _grids(self):
         return (self.x_grid, self.y_grid, self.z_grid)
 
+    @property
+    def cartesian_vertices(self):
+        """Returns vertices in cartesian coordiantes. Identical to ``vertices`` for RegularMesh and RectilinearMesh
```

Comment:
I don't understand the last part -- how can it be identical to `vertices` when it uses `self.vertices` and then transforms it?

Reply:
Good point. Not quite identical. I'll address that.

Reply:
This should probably return the same shape as the ``vertices`` property for consistency. I know @eepeterson has a good rationale for why the shape of the `MeshBase
.vertices` property is what it is. I'll make sure that what I'm adding conforms to that.

Reply:
Okay I've moved these transposes and reshapes into the VTK methods so these external properties will be consistent on the `StructuredMesh` classes.